core/domain/user_jobs_continuous.py

```diff
@@ -418,6 +471,27 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(item):
+        """Implements the map function (generator).
+        Computes exploration data for every contributor and owner of the exploration.
+
+        Args:
+            item: The parameter passed to this function is a single element of
+            the type given by entity_class().
+
+        Returns:
+            This function may yield as many times as appropriate the key/value 2-tuples. 
+            For example, (id, exploration_data) where
+                id: str. The unique id of contributor or owner of the exploration
```

Comment:
Reframe this so that it's more centered on the user, rather than on the exploration. E.g.:

id: str. A user ID.
exploration_data: dict. A dict that includes entries for all the explorations that this user contributes to or owns. Each entry has the following keys: ...